src/Controls/src/Core/Toolbar/Toolbar.Windows.cs

```diff
@@ -65,13 +65,12 @@ internal void UpdateMenu()
 				button.SetAutomationPropertiesName(item);
 				button.SetAutomationPropertiesAccessibilityView(item);
 				button.SetAutomationPropertiesHelpText(item);
-				button.UpdateTextColor(BarTextColor);
-
 				button.SetAutomationPropertiesLabeledBy(item, null);
 
 				ToolbarItemOrder order = item.Order == ToolbarItemOrder.Default ? ToolbarItemOrder.Primary : item.Order;
 				if (order == ToolbarItemOrder.Primary)
 				{
+					button.UpdateTextColor(BarTextColor);
```

Comment:
LGTM, just a pending snapshot for the added test.

Reply:
Added pending snap